crates/rolldown/tests/fixtures/topics/live_bindings/.named_exports_in_common_chunks_cjs/artifacts.snap

```diff
@@ -0,0 +1,90 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+expression: snapshot
+input_file: crates/rolldown/tests/fixtures/topics/live_bindings/named_exports_in_common_chunks_cjs
+---
+# Assets
+
+## async-entry.cjs
+
+```js
+"use strict";
+
+const { count, inc, reset } = require("./shared.cjs");
```

Comment:
This case failed here. We need to use property access instead of cons bindings here to keep bindings live.